modules/test/playwright/tests/portal-workflow-kaleo-designer-web/notification.spec.ts

```diff
@@ -193,3 +210,64 @@ test('create a notification using role type', async ({
+	} as WorkflowDefinition;
```

Comment:
We can use `Partial<WorkflowDefinition>` to avoid set the properties to optional.

Reply:
We decided to set `Partial<WorkflowDefinition>` directly on the expected parameter of the POST method, and the workflow definition info is now being sent through the 'getWorkflowDefinition' function. Thanks!